### ISSUE TITLE
Webhook not working

### ISSUE THREAD

**Issue by derekoharrow** [Sonarr/Sonarr#4679]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Trying to get Sonarr (and Radarr et al) to send a webhook to Home Assistant when there is an error.

I have the webhook setup in home assistant, and can call this using a CURL command such as:

`curl -X POST https://blahblahblahblahblah.ui.nabu.casa/api/webhook/radarr-health
`
This works fine - even from within the docker container that sonarr is running in.

However, when I set this up in the Connect settings in sonarr, it says it's triggered it, but nothing happens - the webhook isn't triggered/received by Home Assistant.

I've also tried doing this locally and calling the local home assistant machine (hass.lan:8123) rather than going via Nabu Casa. Both work locally from the command line and from within the sonarr container, but not from Sonarr itself.

### Expected Behavior

I'd expect Sonarr to trigger Home Assistant by calling the Webhook

### Environment

```markdown
- OS: Ubuntu 20.0.4 LTS
- Sonarr: 3.0.6.1335
- Docker Install: YES
- Using Reverse Proxy:
- Browser: Edge
```

### What branch are you running?

Main

### Anything else?

Log extract showing Sonarr allegedly calling the API, but nothing happens from sonarr (but does from docker command line using curl)

```
2021-10-04 16:16:14.6|Trace|Http|Req: 8277 [POST] /api/v3/notification/test (from ::ffff:172.18.0.1 Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.61 Safari/537.36)
2021-10-04 16:16:14.6|Trace|HttpClient|Req: [POST] https://blahblahblahblahblah.ui.nabu.casa/api/webhook/radarr-health
2021-10-04 16:16:14.6|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2021-10-04 16:16:14.9|Trace|HttpClient|Res: [POST] https://blahblahblahblahblah.ui.nabu.casa/api/webhook/radarr-health: 200.OK (347 ms)
2021-10-04 16:16:14.9|Trace|Http|Res: 8277 [POST] /api/v3/notification/test: 200.OK (358 ms)
2021-10-04 16:16:14.9|Debug|Api|[POST] /api/v3/notification/test: 200.OK (358 ms)
```

**Comment by bakerboy448** [comment]
Sonarr's logs indicate it is being sent and receiving  200 back; so it does not appear to be a sonarr issue, but something on your end.

perhaps not handling the test properly?

Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC ](https://web.libera.chat/?channels=#sonarr)for support/questions.

can reopen if later discovered to be bug
